runners_manager/vm_creation/openstack.py

```diff
@@ -74,6 +75,12 @@ def create_vm(self, runner: Runner, runner_token: int or None,
             userdata=self.script_init_runner(runner, runner_token, github_organization,
                                              installer)
         )
+        while self.nova_client.servers.get(instance.id).status not in ['ACTIVE', 'ERROR']:
+            time.sleep(2)
```

Comment:
Open thought, do you think we should increment the sleep depending on the number of retries?

Reply:
Well, if we want to reduce the number of api call, we could take the average spawn time and then start to check the status. (with an average fo 30 sec, it remove 15 calls already)